tracker/peerstore/redis_test.go

```diff
@@ -14,31 +14,32 @@
 package peerstore
 
 import (
+	"math/rand"
+	"os"
 	"testing"
 	"time"
 
 	"github.com/uber/kraken/core"
 
+	"github.com/alicebob/miniredis"
 	"github.com/andres-erbsen/clock"
-	"github.com/garyburd/redigo/redis"
 	"github.com/stretchr/testify/require"
 )
 
-func redisConfigFixture() RedisConfig {
-	return RedisConfig{
-		Addr:              "localhost:6380",
-		PeerSetWindowSize: 30 * time.Second,
-		MaxPeerSetWindows: 4,
-	}
+func TestMain(m *testing.M) {
+	rand.Seed(0)
```

Comment:
Why is this needed?

Reply:
Thought we would need this for miniredis to have reproducible random numbers but we don't care about that in the tests.
Will remove.